cmd/generate-index/provider.yaml

```diff
@@ -44,7 +44,7 @@ cloud:
     oracle:
       - https://docs.oracle.com/en-us/iaas/tools/public_ip_ranges.json
     azure:
-      - https://download.microsoft.com/download/7/1/D/71D86715-5596-4529-9B13-DA13A5DE5B63/ServiceTags_Public_20221128.json
+      - https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519
```

Comment:
@neonbunny , above link seems to be broken
```console
$  curl -I "https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519" 
HTTP/2 302 
content-length: 0
location: https://www.microsoft.com/en-us/download/404Error.aspx
```

Reply:
```console
curl -I "https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519" 
HTTP/2 200 
cache-control: no-cache, no-store
pragma: no-cache
content-type: text/html
expires: -1
p3p: CP="ALL IND DSP COR ADM CONo CUR CUSo IVAo IVDo PSA PSD TAI TELo OUR SAMo CNT COM INT NAV ONL PHY PRE PUR UNI"
correlationvector: 8/X76BUyNE+cyd0m.1.0
access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept
access-control-allow-methods: GET, POST, PUT, DELETE, OPTIONS
access-control-allow-credentials: true
x-frame-options: SAMEORIGIN
x-sitemuse-origin: Azure
x-azure-ref: 0YmNaZAAAAAAienYLqvQ+QLvUSMRBgm5qU0cyRURHRTI4MjIAMWIyYjIyYzEtY2IwYy00OTUwLTg4NzktZDIxMmYyYWM1YTRi
x-edgeconnect-midmile-rtt: 65
x-edgeconnect-origin-mex-latency: 247
date: Tue, 09 May 2023 15:14:42 GMT
set-cookie: MS-CV=8/X76BUyNE+cyd0m.1; domain=.microsoft.com; expires=Wed, 10-May-2023 15:14:42 GMT; path=/;samesite=None; secure
tls_version: tls1.3
strict-transport-security: max-age=31536000
ms-cv: CASMicrosoftCV2e48ec6b.0
ms-cv-esi: CASMicrosoftCV2e48ec6b.0
x-rtag: SMAPROD
```

works for me as well.

Reply:
not sure why it was not working earlier . it works now
```console
$ curl -I "https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519" 
HTTP/2 200 
content-type: text/html
cache-control: no-cache, no-store
pragma: no-cache
expires: -1
p3p: CP="ALL IND DSP COR ADM CONo CUR CUSo IVAo IVDo PSA PSD TAI TELo OUR SAMo CNT COM INT NAV ONL PHY PRE PUR UNI"
correlationvector: yZ9QOAtzd0iy98MQ.1.0
access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept
access-control-allow-methods: GET, POST, PUT, DELETE, OPTIONS
access-control-allow-credentials: true
x-frame-options: SAMEORIGIN
x-sitemuse-origin: Azure
x-azure-ref: 20230526T204913Z-macu5fq3rh3vh9h4quvew7vezc000000009g00000000eeyn
x-edgeconnect-midmile-rtt: 56
x-edgeconnect-origin-mex-latency: 284
date: Fri, 26 May 2023 20:49:13 GMT
set-cookie: MS-CV=yZ9QOAtzd0iy98MQ.1; domain=.microsoft.com; expires=Sat, 27-May-2023 20:49:13 GMT; path=/;samesite=None; secure
tls_version: tls1.3
strict-transport-security: max-age=31536000
ms-cv: CASMicrosoftCV216a2135.0
ms-cv-esi: CASMicrosoftCV216a2135.0
x-rtag: SMAPROD

```